docs/markdown/Comparisons.md

```diff
@@ -14,7 +14,7 @@ Excellent support for legacy Unix platforms, large selection of existing modules
 
 ### Cons ###
 
-Needlessly slow, complicated, hard to use correctly, unreliable, painful to debug, ununderstandable for most people, poor support for non-Unix platforms (especially Windows).
```

Comment:
The choice of the word _ununderstandable_ is a conscious stylistic choice. If Markdown supports comments it might make sense to mention that here.